apps/services/auth/ids-api/src/app/delegations/test/delegations-filters-types.ts

```diff
@@ -62,23 +67,29 @@ export class TestCase {
     this.protectedScopes = options.protectedScopes ?? []
     this.scopeAccess = options.scopeAccess ?? []
     this.expectedFrom = options.expectedFrom
+    this.expectedTypes = options.expectedTypes
   }
 
-  get domain(): CreateDomain {
-    return { name: this.domainName }
+  get domains(): CreateDomain[] {
+    const domains = [{ name: this.domainName }]
+    if (this.otherDomainName) {
+      domains.push({ name: this.otherDomainName })
+    }
+    return domains
   }
 
   get apiScopes(): CreateApiScope[] {
     return this.scopes.map((s) => ({
       name: s,
       description: s,
       displayName: s,
-      domainName: this.domainName,
+      domainName: s.includes('-od') ? this.otherDomainName : this.domainName,
```

Comment:
Shouldn't `'-od'` be in a variable with a descriptive name?

Reply:
Fixed.